lib/board.rb

```diff
@@ -0,0 +1,56 @@
+require 'pry'
+class Board
+  attr_accessor :cells, :board, :token
+
+# Board object = #<Board:0x000000018f0d30 @cells=[" ", " ", " ", " ", " ", " ", " ", " ", " "]>
+# Each method calls on the board object and analyzes it based on an input
+
+  def initialize
+    @cells = Array.new(9, " ")
+    #create the board self and populates the @cells
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.